scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -780,16 +782,121 @@ void BPF_STRUCT_OPS(layered_enqueue, struct task_struct *p, u64 enq_flags)
 			return;
 	}
 
-	bpf_for(idx, 0, nr_possible_cpus) {
-		s32 cand = (preempt_cursor + idx) % nr_possible_cpus;
+	if (!disable_topology) {
+		if (!(cachec = lookup_cache_ctx(cctx->cache_idx)) ||
+		    !(nodec = lookup_node_ctx(cctx->node_idx)))
+			return;
```

Comment:
nit: would it be clearer if we split these into two early returns?
```suggestion
                if (!(cachec = lookup_cache_ctx(cctx->cache_idx)))
                        return;
		if (!(nodec = lookup_node_ctx(cctx->node_idx)))
			return;
```